scripts/players_import.js

```diff
@@ -0,0 +1,102 @@
+var common = require('../routes/common')
+var config = common.config();
+var express = require('express');
+var router = express.Router();
+var parseString = require('xml2js').parseString;
+var http = require("http"),
+    async = require('async'),
+    mongojs = require("mongojs"),
+
+    db = mongojs.connect(config.mongo_uri);
+
+
+var request = require('request');
+var difference = require('array-difference');
+
+var players_model = require('../models/players.js');
+
+//process.argv grabs the command line arguments
+var league = process.argv[2];
+var options = {};
+options.only_insert = true;
+var rosters = [];
+
+    //fetch the team collections
+    db.collection('teams').find({'league': league}).toArray(function (err, teams) {
+      //fetch the players collectiom
+      db.collection('players').find().toArray(function (err, players){
+        var common_team_ids = [];
+        var all_team_ids = [];
+        //loop through and filter out the players collections that we already have
```

Comment:
But player info changes. Don't we want to check for last updated dates and filter on that, instead of simple existence?